third-party/py/pex/pex/pex.py

```diff
@@ -104,6 +104,9 @@ def _site_libs(cls):
       site_libs = set()
     site_libs.update([sysconfig.get_python_lib(plat_specific=False),
                       sysconfig.get_python_lib(plat_specific=True)])
+    # On windows getsitepackages() returns the python stdlib too.
+    if sys.prefix in site_libs:
+      site_libs.remove(sys.prefix)
```

Comment:
I think the crash you saw was an extreme example of the bug - the windows box I used to test doesn't have any packages installed, so the actual thing I saw was an `ImportError` trying to import `json`. To trace it down further I set PEX_VERBOSE=5 and saw that the stdlib was being "scrubbed" from the path.